client/src/components/header/Header-nav/HeaderNav.jsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styles from './HeaderNav.module.scss';
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import { NavLink, useHistory } from 'react-router-dom';
+import { Button } from '@material-ui/core';
+import avatarLogo from '../../../assets/img/avatar.jpg';
+import { API_URL } from '../../../config';
```

Comment:
Do not import such things into markup file. Better create helper function which knows how to build urls, in utils or store folder.